src/EditorControls.js

```diff
@@ -30,6 +30,7 @@ class EditorControls extends Component {
       dataSourceValueRenderer: this.props.dataSourceValueRenderer,
       dataSourceOptionRenderer: this.props.dataSourceOptionRenderer,
       dictionaries: this.props.dictionaries || {},
+      frames: gd._transitionData ? gd._transitionData._frames : [],
```

Comment:
although here I've noticed it's a little odd that my gd doesn't always get _transitionData, sometimes it's undefined..
it's a little odd because it seems to me like this key is always on gd, even if there's no frames...

like here:https://codepen.io/veraz/pen/zRRONV